src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,6 @@
+$day-width: 100px;
+$day-height: 100px;
+$calendar-padding: 10px;
+$color-day-background: #eee;
+$color-day-border: black;
+$day-in-month: 29;
```

Comment:
The variable $day-in-month seems to be unused. Please remove it if it's not needed or use it in your code if necessary.